simpa/utils/libraries/molecule_library.py

```diff
@@ -71,24 +68,6 @@ def update_internal_properties(self, settings):
             self.internal_properties[Tags.DATA_FIELD_ALPHA_COEFF] += molecule.volume_fraction * \
                 molecule.alpha_coefficient
 
-        if self.filler is not None:
-            filling_volume_fraction = 1.0 - self.internal_properties.volume_fraction
-            if (filling_volume_fraction < 0.0).any():
-                raise AssertionError("The total volume fraction defined was larger than 1.0."
-                                     "Please double check the bounds of the molecular compositions.")
```

Comment:
Apparently we tested "total volume > 1" only if a filler was defined.
It is for sure worth, as you mentioned, to add a check for that!